tests/scale-tests/harness/harnessctl/install_components.go

```
@@ -0,0 +1,379 @@
+		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: cfg.NVSNamespace},
+		Type:       corev1.SecretTypeOpaque,
+		StringData: map[string]string{"oidc-client-secret": "placeholder"},
+	}, metav1.CreateOptions{})
+	return err
+}
+
+// unlockPendingHelmRelease rolls back a stuck pending-* release.
+func (c *clients) unlockPendingHelmRelease(ctx context.Context, release, ns string) error {
+	status, err := helmStatus(ctx, release, ns)
+	if err != nil || status == "" {
+		return nil // no release yet
```

golangci-lint (2.12.2)

[error] error is not nil (line 328) but it returns nil

(nilerr)